migrations/10.0.md

```diff
@@ -0,0 +1,42 @@
+# Major 10.0 Migration guide
+> [!IMPORTANT]
+> Please make sure to first follow the ["Upgrade new version guide"](https://github.com/AmadeusITGroup/otter/blob/release/main/docs/core/UPGRADE_NEW_VERSION.md) before going through these steps.
+
+
+## Personalization
+Customization now only relies on the ```@O3rComponent``` decorator.<br>
+The following interfaces and decorator (already deprecated) have been fully removed.
+* ``@InputMerge`` decorator
```

Comment:
Maybe we could have a paragraph that referenced all removed methods, interfaces, decorators, ...
Because I'm not sur `@InputMerge` is linked to personalization
(and we can remove it from authorized decorator in the eslint config `packages/@o3r/eslint-config-otter/rules/typescript/eslint.cjs L85`